plutus-playground/README.md

```diff
@@ -32,12 +32,27 @@ Then run: `yarn run webpack` for a production build on http://localhost:8080
 You may also want to run `yarn run purs:ide` to start `psc-ide`
 support running with the correct paths.
 
-### nix
+## nix
+
+Fair warning before we start: You may struggle with PureScript on Nix on OSX.
+
+The client and server can be built from the top-level of this repo with:
+
+```sh
+nix-build \
+  --option trusted-public-keys "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" \
+  --option substituters https://hydra.iohk.io \
```

Comment:
You really don't need to keep putting these options in if you take the advice in the top-level README and put them in your `nix.conf`.